src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SuppressFormattingRule.cs

```diff
@@ -375,6 +375,12 @@ private static void AddInitializerSuppressOperations(List<SuppressOperation> lis
                 return;
             }
 
+            if (node is CollectionExpressionSyntax collectionExpression)
+            {
+                AddSuppressAllOperationIfOnMultipleLine(list, collectionExpression.OpenBracketToken, collectionExpression.CloseBracketToken);
+                return;
+            }
+
```

Comment:
just a move for clarity.  this puts collection expression suppression next to initializer-expression suppression.